docs/trulens_explain/attribution_parameterization.md

```diff
@@ -96,47 +100,47 @@ If you want to explain an internal layer, you do not move the DoI layer.
 **Use Case:** in NLP, baselines are tokens, but the interpolation is on the embedding layer.  
 **Cuts needed:** Baseline different from DoI.  
 **Model inputs** → InputCut  
-**Baseline Cut** →  tokens, anchor:'out'  
-**DoI/Attribution Cut** → embeddings, anchor:'out'  
+**Baseline Cut** → Tokens, anchor:'out'  
+**DoI/Attribution Cut** → Embeddings, anchor:'out'  
 **QoI Cut** → OutputCut
  
-#### Case 7: Putting it together - The most complex one one we can do with our Trulens
+#### Case 7: Putting it together - The most complex case we can perform with TruLens
 
 **Use Case:** Internal layer explanations of NLP, on the logit layer of a model with probit outputs.  
 **Model inputs** → InputCut  
-**Baseline Cut** → tokens, anchor:'out'  
-**DoI Cut** → embeddings, anchor:'out'  
+**Baseline Cut** → Tokens, anchor:'out'  
+**DoI Cut** → Embeddings, anchor:'out'  
 **Attribution Cut** → Internal layer, anchor:'out'  
-**QoI Cut** → logit layer, anchor:'out'
+**QoI Cut** → Logit layer, anchor:'out'
  
 ### Summary
 
 **InputCut** is model args / kwargs.  
 **OutputCut** is the model output.
 
-**Baseline Cut** is the tensor associated with the Integrated gradients baseline. Can be the InputCut or later.  
+**Baseline Cut** is the tensor associated with the Integrated Gradients baseline. Can be the InputCut or later.  
 **DoI Cut** is the tensor associated with explanation sampling. Can be the BaselineCut or later.  
 **Attribution Cut** is the tensor that should be explained. Can be the DoICut or later.  
-**QoI Cut** is what is being explained with a QoI. Must be after the AttributionCut.  
+**QoI Cut** is what is being explained with a QoI. Must be after the AttributionCut.
 
 ### Advanced Definitions
 
 **What is a Distribution of Interest (DoI)?**
 
 The distribution of interest is a concept of aggregating attributions over a sample or distribution. 
 
-* GradCam does this over a guassian distribution of inputs. 
-* Shapley values do this over different background data. 
-* Integrated gradients does this over an interpolation from a baseline to the input.
+* [Grad-CAM](https://ieeexplore.ieee.org/document/8237336) does this over a Gaussian distribution of inputs. 
+* [Shapley values (SHAP)](https://shap.readthedocs.io/en/latest/example_notebooks/overviews/An%20introduction%20to%20explainable%20AI%20with%20Shapley%20values.html) do this over different background data. 
+* [Integrated Gradients](https://arxiv.org/abs/1703.01365) do this over an interpolation from a baseline to the input.
```

Comment:
Looking for feedback on specific references to point to here. Is it more digestable to point to implementations/tutorials/packages, to academic papers, or to both?

Reply:
Both is my vote. Maybe in parenthesis like: Grad-CAM (Paper, Implementation/Tutorial/Package)?